app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,18 @@
     <string name="title_load_failure">Load data failure</string>
     <string name="title_no_data">No data!</string>
     <string name="title_find_place">Find place</string>
+    <string name="title_dew_point">Dew point</string>
+    <string name="title_uv_index">UV Index</string>
+    <string name="title_location">Location</string>
+    <string name="title_activity_select_date">SelectDateActivity</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_activity_select_date` appears to be unused
 Unused resources make applications larger and slow down builds.